packages/upstream/nocrypto.0.5.4+2/opam

```diff
@@ -35,13 +34,19 @@ depends: [
   ("mirage-solo5" & "mirage-entropy" & "zarith-freestanding")
 ]
 conflicts: [
-  "topkg" {<"0.8.0"}
+  "topkg" {<"0.9.1"}
   "ocb-stubblr" {<"0.1.0"}
   "mirage-xen" {<"2.2.0"}
   "sexplib" {="v0.9.0"}
 ]
+
 patches: [
-    "0001-Update-to-support-latest-cstruct-and-ppx_sexp_conv-a.patch"
+    "0001-add-missing-runtime-dependencies-in-_tags.patch"
+    "0002-add-ppx_sexp_conv-as-a-runtime-dependency-in-the-pac.patch"
+    "0003-Auto-detect-ppx_sexp_conv-runtime-library.patch"
+    "0004-pack-package-workaround-ocamlbuild-272.patch"
+    "0005-use-modern-cstruct-findlib.patch"
+    "0006-explicit-dependency-on-sexplib.patch"
```

Comment:
This package has accumulated quite a few packaging patches, looks like upstream hasn't made a release (or merged PRs) in quite a while

Reply:
While the latest release was done in 2017, master was last updated on February. The PR for porting the build system to dune got stuck on early April: https://github.com/mirleft/ocaml-nocrypto/pull/158